src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsEmailPreferences/SettingsEditSettingsEmailPreferences.tsx

```diff
@@ -28,7 +28,7 @@ export const SettingsEditSettingsEmailPreferences: React.FC<SettingEditSettingsE
 
   const handleSelect = async newEmailFrequency => {
     const variables = { emailFrequency: newEmailFrequency }
-    try{
+    try {
```

Comment:
How is code without prettier making it into the codebase?

Reply:
A lot of folks `--no-verify` and don't use VSCode (or hate prettier). But this is the minority, so its a bit heavy handed to fail CI on a prettier check.